shortcut_composer/core_components/controllers/core_controllers.py

```diff
@@ -65,6 +66,28 @@ def get_label(self, value: Tool) -> QIcon:
         return value.icon
 
 
+class ColorSamplerBlendController(Controller):
+    """
+    Gives access to Blend option in Color Sampler tool.
+    """
+
+    default_value: int = 100
+
+    def __init__(self) -> None:
+        self.blend_spinbox_finder = ColorSamplerOptionsFinder()
+    
+    def get_value(self) -> int:
+        """Get current blend percentage."""
+        return self.blend_spinbox_finder.get_blend()
+    
+    def set_value(self, blend: int) -> None:
+        """Set a passed blend percentage."""
+        self.blend_spinbox_finder.set_blend(blend)
+    
+    def get_label(self, value: int) -> Text:
+        return Text(f"{value} %", Colorizer.percentage(value))
```

Comment:
Wywaliłbym spację między {value} a %.

Wiem, że u Ciebie te labele mieszczą dużo znaków, bo zmieniłeś czcionkę w ustawieniach, ale to jest bug - jak go naprawię, cztery znaki powinny być maksem.

Reply:
Ah, szkoda, bo kiepsko mi się czyta liczby jak % jest doklejone, ale rozumiem i zmienię :) Może ogarnę jak zrobić % trochę ciemniejszy od liczby... hmmmm.